ChangeLog

```diff
@@ -132,6 +133,10 @@ Changes
    * Only run AES-192 self-test if AES-192 is available. Fixes #963.
    * Tighten the RSA PKCS#1 v1.5 signature verification code and remove the
      undeclared dependency of the RSA module on the ASN.1 module.
+   * Update all internal usage of deprecated message digest functions to the
+     new ones with return codes. In particular, this modifies the
+     mbedtls_md_info_t structure. Propagate errors from these functions
+     everywhere except some locations in the ssl_tls.c module.
```

Comment:
This sounds a bit strange and inaccurate to me from a linguistic point of view - "everywhere except some". Maybe "In most locations except several in the ssl_tls.c module" would be better?

Reply:
It doesn't sound strange to me, but I'll ask a third opinion.

Reply:
I would have expected `... everywhere except for some locations ...`. @Patater, we need your assessment as a natural speaker.

Reply:
What @hanno-arm expects is what I'd expect, but the old way is not strictly wrong. No change necessary here.